packages/libxcb.rb

```diff
@@ -26,7 +26,7 @@ class Libxcb < Package
   depends_on 'pthread_stubs'
 
   def self.build
-    system "python=/usr/local/bin/python2.7 ./configure"
+    system "python=/usr/local/bin/python2.7 ./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
```

Comment:
system "python=#{CREW_PREFIX}/bin/python2.7 ... ?

Reply:
It can be removed.